src/dmtest/vdo/dedupe_tests.py

```diff
@@ -0,0 +1,91 @@
+from dmtest.assertions import assert_raises, assert_equal, assert_near
+from dmtest.vdo.utils import standard_stack, standard_vdo
+import dmtest.fs as fs
+import dmtest.process as process
+import dmtest.units as units
+import dmtest.vdo.stats as stats
+import dmtest.test_register as reg
+
+import os
+import re
+import threading
+import logging as log
+import time
+
+#---------------------------------
+
+def make_delta_stats(stats_post, stats_pre):
+    """
+    Given two stats dictionaries, the code recursively replaces all int
+    fields in the stats_post dictionary with the delta value between post
+    and pre.
```

Comment:
As I read the code, it's not replacing anything in the stats_post dictionary, it's creating and returning a new one showing the differences. (What does it do for non-int, non-dict fields?)